code/modules/clothing/head/helmet.dm

```diff
@@ -72,3 +73,65 @@
 	cold_protection = HEAD
 	min_cold_protection_temperature = SPACE_HELMET_MIN_COLD_PROTECTION_TEMPERATURE
 	siemens_coefficient = 0.5
+
+/obj/item/clothing/head/helmet/lightriot
+	name = "light riot helmet"
+	desc = "Standart issue Ironhammer helmet. Basic hud and targeting system included."
+	icon_state = "lightriot"
+	body_parts_covered = HEAD|FACE
+	armor = list(melee = 60, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0)
+	flags_inv = HIDEEARS
+	siemens_coefficient = 0.7
+	flash_protection = FLASH_PROTECTION_MAJOR
+	action_button_name = "Toggle Security Hud"
+	var/obj/item/clothing/glasses/hud/security/hud
+
+/obj/item/clothing/head/helmet/lightriot/New()
+	..()
+	hud = new(src)
+	hud.canremove = 0
+
+/obj/item/clothing/head/helmet/lightriot/ui_action_click()
+	toggle()
+
+/obj/item/clothing/head/helmet/lightriot/verb/toggle()
+	set name = "Toggle Security Hud"
+	set desc = "Shows you jobs and criminal statuses"
+	set category = "Object"
+	if(usr.stat || usr.restrained())
+		return
+	var/mob/user = usr
```

Comment:
Please specify and check full human type path to avoid possible runtime errors and bugs.